### DISCUSSION TITLE
SSO auth

### DISCUSSION THREAD

**Discussion by dominhhai** [medusajs/medusa#2177] [Q&A] [answered]
How can I login by SSO such as Twitter or AWS Cognito?

**Comment by srindom** [comment, marked as answer]
This is not supported at the moment, but we plan to add SSO in the future.

**Comment by sitewerk-digital-solutions** [comment]
@srindom is there plans when this feature will be released?
Your product is sick but this feature is crucial for many of our projects. (we've been evaluating a new shop system which works along the strapi CMS :) )
